### DISCUSSION TITLE
Advanced Topics Kick-off!

### DISCUSSION THREAD

**Discussion by fboucher** [microsoft/Power-Platform-For-Beginners#3] [Advanced Topics and Applications]
Do you have a section of the [template](https://github.com/microsoft/Power-Platform-For-Beginners/tree/main/templates) you would like to start working on?
Let your teammates know!

Then you can **Fork** the repo and start creating content!

**Comment by LuiseFreese** [comment]
Hey there  :-)

**Reply by fboucher** [reply]
Hey @LuiseFreese 👋
Any thoughts on how you would teach Advanced Topics using a project-based lesson? What kind of project/ story would you use in a step-by-step approach that would help people to understand those concepts?

**Reply by julians1966** [reply]
Hi both.
The Advanced topics is quite a variation of topics. I guess we will not be going into detail so will focus on concepts so using scenarios to explain the concepts and decisions might be the bess way

**Reply by julians1966** [reply]
I am not sure that projects e.g., creating PCF controls is relevant for beginners, is the right way to go

**Reply by fboucher** [reply]
@julians1966 @LuiseFreese you are the author/ creator... Listening to your ideas!
I was just trying to get the conversion going :)

**Comment by fboucher** [comment]
I'll close this discussion now that most happens on Teams and keep only the more general ones open for now. :)
